### PR TITLE
Add pylint config and update readme

### PR DESCRIPTION
**Pull request by yf1357**
1. add pylint.config
2. update readme.md
3. change backref in models.py

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,16 +17,32 @@ Then all you have to do is clone this repo and invoke vagrant:
...
+ 
+ DELETE http://localhost:5000/wishlists/{wishlistID} - Delete a wishlist record in the database  
+ 
+ GET http://localhost:5000/wishlists?name="wishlist_1" - Query the database by the name of the wishlist   
+
+ PUT http://localhost:5000/wishlists/{wishlistId}/disabled - It disables the target wishlist
+ 
+ PUT http://localhost:5000/wishlists/{wishlistId}/enabled - It enables the target wishlist
+ 
+ ## Manually running the Tests
```
Comment (abhinavGupta16) [inline README.md:49]: This is nice. Thank you for adding this. We have features for CRUD operations on items as well.
/wishlists/<int:wishlist_id>/items
/wishlists/<int:wishlist_id>/items/<int:item_id> etc
Let's add them too!

**Review by abhinavGupta16 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yf1357) [reply README.md:49]: Add create and get items in Read.md. It seems we don't have list and delete now. Please check again.

Comment (abhinavGupta16) [reply README.md:49]: We can add them when #10 is raised.

**Review by abhinavGupta16 — APPROVED** [review]

**Review by Jesse1204 — APPROVED** [review]

File: service/models.py [hunk 2]
```diff
@@ -177,7 +177,7 @@ def __str__(self):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(63), nullable=False)
     user_id = db.Column(db.Integer, nullable=False)
-    items = db.relationship('Item', backref='account', lazy=True)
+    items = db.relationship('Item', backref='wishlists', lazy=True)
```
Comment (xiafanzeng) [inline service/models.py:180]: should be 'wishlist' rather than 'wishlists'

**Pull request merged**